whipper/test/test_result_logger.py

```diff
@@ -141,7 +141,7 @@ def testLogger(self):
             actualLines[0],
             re.compile((
                 r'Log created by: whipper '
-                r'[\d]+\.[\d]+.[\d]+\.dev[\w\.\+]+ \(internal logger\)'
+                r'[\d]+\.[\d]+.[\d]+(\.dev[\w\.\+]+)? \(internal logger\)'
```

Comment:
I’m wondering if the initial rule isn’t missing a `\` before the second `.`…

Reply:
It is. (But that should be a separate PR.)

Reply:
Addressed it in a separate commit (f740a0ef0d4503b16485b417d1222032a20cd4b6) and rebased this branch on develop.